### PR TITLE
SIR-3056 Assets V2 Rest API + fix some internal links + estimation

### PR DESCRIPTION
**Pull request by waynesayonara**

### CODE REVIEW

File: source/includes/v2/_assets.md [hunk 1]
```diff
@@ -30,33 +35,42 @@ var request = new AssetSearchRequest
...
-name | type | description | example
----- | ---- | ----------- | -------
-`Id` | *optional*, *long* | ID of the asset to search | 100553
-`BlockchainId` | *optional*, *string* | Exact blockchain id to search | bitcoin-private
-`Symbol` | *optional*, *string* | Text to search in the asset symbol | BTC
-`TokenId` | *optional*, *string* | Text to search in the asset symbol | BTC
-`Address` | *optional*, *string* | Exact address to search | 0xC1701AbD559FC263829CA3917d03045F95b5224A
-`Accuracy` | *optional*, *int* | Exact accuracy to search | 8
-`Pagination` | *optional*, *[Pagination](#api-usage-pagination-request)* | |
+| name           | type                                                                  | description                                          | example                                    |
```
Comment (KonstantinRyazantsev) [inline source/includes/v2/_assets.md:45]: rest/gRPC field names difference is not reflected here

File: source/includes/v2/_assets.md [hunk 2]
```diff
@@ -89,72 +103,107 @@ var request = new AssetAddAttributesRequest
...
+
 ### Request
 
-name | type | description | example
----- | ---- | ----------- | -------
-`IdempotencyId` | *string* | Request unique identifier | "AD1D5199-1D1F-4B2C-AE6C-DED0DAAC4D34"
-`AssetId` | *long* | Asset identifier in Sirius | 10001
-`Aml` | *[AmlAssetAttributesCreateInfo](#assets-models-amlassetattributescreateinfo)* | AML asset attributes | 
-`Brokerage` | *[BrokerageAssetAttributesCreateInfo](#assets-models-brokerageassetattributescreateinfo)* | Brokerage asset attributes |
+| name               | type                                                                                      | description                | example                                |
```
Comment (KonstantinRyazantsev) [inline source/includes/v2/_assets.md:141]: and here

File: source/includes/v2/_assets.md [hunk 3]
```diff
@@ -89,72 +103,107 @@ var request = new AssetAddAttributesRequest
...
-`Symbol` | *string* | Asset symbol | BTC
-`TokenId` | *string* | Token ID | 101
-`Address` | *string* | Exact address | 0xC1701AbD559FC263829CA3917d03045F95b5224A
-`Accuracy` | *int* | Exact accuracy | 8
-`Name` | *string* | The name of token | Toy
-`MetadataUrl` | *string* | The token metadata URL | ipfs://metadata.json
-`Type` | *string* | The blockchain-specific type of token | ERC-20
-`Aml` | *[AmlAssetAttributesInfo](#assets-models-amlassetattributesinfo)* | The AML attributes |
-`Brokerage` | *[BrokerageAssetAttributesInfo](#assets-models-brokerageassetattributesinfo)* | The brokerage attributes |
+| name           | type                                                                          | description                           | example                                    |
```
Comment (KonstantinRyazantsev) [inline source/includes/v2/_assets.md:167]: and here

File: source/includes/v2/_assets.md [hunk 4]
```diff
@@ -89,72 +103,107 @@ var request = new AssetAddAttributesRequest
...
+| `Aml`          | *[AmlAssetAttributesInfo](#assets-models-amlassetattributesinfo)*             | The AML attributes                    |                                            |
+| `Brokerage`    | *[BrokerageAssetAttributesInfo](#assets-models-brokerageassetattributesinfo)* | The brokerage attributes              |                                            |
 
 ### AmlAssetAttributesCreateInfo
 
-name | type | description | example
----- | ---- | ----------- | -------
-`Chainalysis` | *[AmlChainalysisAssetAttributesInfo](#assets-models-amlchainalysisassetattributesinfo)* | Chainalysis attributes |
-`MerkleScience` | *[AmlMerkleScienceAssetAttributesInfo](#assets-models-amlmerklescienceassetattributesinfo)* | MerkleScience attributes |
+| name            | type                                                                                        | description              | example |
```
Comment (KonstantinRyazantsev) [inline source/includes/v2/_assets.md:183]: and here

File: source/includes/v2/_assets.md [hunk 5]
```diff
@@ -89,72 +103,107 @@ var request = new AssetAddAttributesRequest
...
 ### AmlAssetAttributesInfo
 
-name | type | description | example
----- | ---- | ----------- | -------
-`Id` | *long* | ID of the info | 300001
-`Chainalysis` | *[AmlChainalysisAssetAttributesInfo](#assets-models-amlchainalysisassetattributesinfo)* | Chainalysis attributes |
-`MerkleScience` | *[AmlMerkleScienceAssetAttributesInfo](#assets-models-amlmerklescienceassetattributesinfo)* | MerkleScience attributes |
-`CreatedAt` | *DateTime* | Create date time |
-`UpdatedAt` | *DateTime* | Update date time |
+| name            | type                                                                                        | description              | example |
```
Comment (KonstantinRyazantsev) [inline source/includes/v2/_assets.md:190]: and here

File: source/includes/v2/_assets.md [hunk 6]
```diff
@@ -89,72 +103,107 @@ var request = new AssetAddAttributesRequest
...
 ### AmlChainalysisAssetAttributesInfo
 
-name | type | description | example
----- | ---- | ----------- | -------
-`Symbol` | *string* | Chainalysis asset symbol | BTC
-`IsCheckWithdrawalsEnabled` | *bool* | Indicates that withdrawals check enabled | true
-`IsCheckDepositsEnabled` | *bool* | Indicates that deposits check enabled | true
-`IsNotifyWithdrawalsEnabled` | *bool* | Indicates that notify withdrawals check enabled | true
-`TransferFormat` | *[ChainalysisTransferFormat](#assets-models-chainalysistransferformat-enum)* | Update date time |
+| name                         | type                                                                         | description                                     | example |
```
Comment (KonstantinRyazantsev) [inline source/includes/v2/_assets.md:200]: and here

File: source/includes/v2/_assets.md [hunk 7]
```diff
@@ -164,25 +213,25 @@ name | type | description | example
 
 ### AmlMerkleScienceAssetAttributesInfo
 
-name | type | description | example
----- | ---- | ----------- | -------
-`Symbol` | *string* | MerkleScience asset symbol | BTC
-`Currency` | *int* | MerkleScience currency ID | 12
-`IsDisabled` | *bool* | Indicates that asset check enabled | true
+| name         | type     | description                        | example |
```
Comment (KonstantinRyazantsev) [inline source/includes/v2/_assets.md:216]: and here

File: source/includes/v2/_assets.md [hunk 8]
```diff
@@ -164,25 +213,25 @@ name | type | description | example
...
+| `Currency`   | *int*    | MerkleScience currency ID          | 12      |
+| `IsDisabled` | *bool*   | Indicates that asset check enabled | true    |
 
 ### BrokerageAssetAttributesCreateInfo
 
-name | type | description | example
----- | ---- | ----------- | -------
-`MinDepositThreshold` | *decimal* | Minimum deposit threshold | 0.0001
-`MinWithdrawalAmount` | *decimal* | Minimum withdrawal amount | 0.1
+| name                  | type      | description               | example |
```
Comment (KonstantinRyazantsev) [inline source/includes/v2/_assets.md:224]: and here

File: source/includes/v2/_assets.md [hunk 9]
```diff
@@ -164,25 +213,25 @@ name | type | description | example
...
 ### BrokerageAssetAttributesInfo
 
-name | type | description | example
----- | ---- | ----------- | -------
-`Id` | *long* | ID of the info | 300001
-`MinDepositThreshold` | *decimal* | Minimum deposit threshold | 0.0001
-`MinWithdrawalAmount` | *decimal* | Minimum withdrawal amount | 0.1
-`CreatedAt` | *DateTime* | Create date time |
-`UpdatedAt` | *DateTime* | Update date time |
+| name                  | type       | description               | example |
```
Comment (KonstantinRyazantsev) [inline source/includes/v2/_assets.md:231]: and here

File: source/includes/v2/_assets.md [hunk 10]
```diff
@@ -6,10 +6,15 @@ The asset is a generic model which represents a native asset, fungible token or
 It's possible that in the different blockchains assets with the same symbol and/or address are existed, so you have to rely only on the asset ID if you need to deterministically identify it.
 If you've received a deposit of an asset which is not supported by Sirius yet, you can add this particular asset to your personal list by specifying all needed asset parameters.
 
+**Rest API:** `POST /api/v2/assets`
+
+**gRPC API:** `swisschain.sirius.api.v2.assets`
+
```
Comment (KonstantinRyazantsev) [inline source/includes/v2/_assets.md:12]: Is this related to entire assets v2 service/controller? If so, why `POST` then?

Comment (waynesayonara) [reply source/includes/v2/_assets.md]: `+`

File: source/includes/v2/_assets.md (hunk 9 above)
Comment (waynesayonara) [reply source/includes/v2/_assets.md]: `+`

File: source/includes/v2/_assets.md (hunk 8 above)
Comment (waynesayonara) [reply source/includes/v2/_assets.md]: `+`

File: source/includes/v2/_assets.md (hunk 7 above)
Comment (waynesayonara) [reply source/includes/v2/_assets.md]: `+`

File: source/includes/v2/_assets.md (hunk 6 above)
Comment (waynesayonara) [reply source/includes/v2/_assets.md]: `+`

File: source/includes/v2/_assets.md (hunk 5 above)
Comment (waynesayonara) [reply source/includes/v2/_assets.md]: `+`

File: source/includes/v2/_assets.md (hunk 3 above)
Comment (waynesayonara) [reply source/includes/v2/_assets.md]: `+`

File: source/includes/v2/_assets.md (hunk 2 above)
Comment (waynesayonara) [reply source/includes/v2/_assets.md]: `+`

File: source/includes/v2/_assets.md (hunk 1 above)
Comment (waynesayonara) [reply source/includes/v2/_assets.md]: `+`

**Review by KonstantinRyazantsev — APPROVED** [review]
